source/manifest.json

```diff
@@ -57,6 +63,9 @@
 		}
 	],
 	"web_accessible_resources": [
-		"assets/resolve-conflicts.js"
-	]
+    {
+      "resources": [ "assets/resolve-conflicts.js" ],
+      "matches": [ "https://*/*" ]
```

Comment:
Also verified to work on https://github.com/refined-github/sandbox/pull/82/conflicts

<img width="673" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/74099612-73ad-45b8-a460-a6183a43acae">